templates/user/my_donation.html.twig

```diff
@@ -25,17 +41,31 @@
                     <th>
                         Montant
                     </th>
+                    <th>
+                        Type
+                    </th>
                 </tr>
             </thead>
             <tbody>
-                {% for donation in donations|reverse %}
+                {% for donation in donations %}
```

Comment:
No message is needed if the adherent don't have any donation ?

Reply:
Not bad idea ^^,